examples/xml-json-conversion/xml_json_conversion.out

```diff
@@ -1,3 +1,3 @@
 bal run xml_json_conversion.bal
-<Store id="AST"><name>Anne</name><address><street>Main</street><city>94</city></address><codes>4</codes><codes>8</codes></Store>
-{"Store":{"name":"Anne","address":{"street":"Main","city":"94"},"codes":["4","8"],"@id":"AST"}}
+<root><Store id="AST"><name>Anne</name><address><street>Main</street><city>94</city></address><codes>4</codes><codes>8</codes></Store></root>
+{"root":{"Store":{"name":"Anne","address":{"street":"Main","city":"94"},"codes":["4","8"],"@id":"AST"}}}
```

Comment:
why do we need another root tag here? can't we consider `Store` as the root tag?